ccd-definitions/.gitignore

```diff
@@ -29,4 +29,13 @@ yarn-error.log
 
 ### Generated CCD Json Definition ###
 ccd-definition
-definitions/development
```

Comment:
Do you need to version the json at all? could you generate it into the build folder?